test/jasmine/tests/image_test.js

```diff
@@ -100,6 +100,62 @@ describe('image supplyDefaults', function() {
     });
 });
 
+describe('image smart layout defaults', function() {
+    var gd;
+    beforeEach(function() {
+        gd = createGraphDiv();
+    });
+
+    afterEach(destroyGraphDiv);
+
+    it('should reverse yaxis if only images are present', function(done) {
+        Plotly.newPlot(gd, [{type: 'image', z: [[[255, 0, 0]]]}])
```

Comment:
Non-blocking. Using `newPlot` is fine, but I would prefer making these pure `supplyDefaults` tests for speed.

Reply:
Ok so I tried to transform a test from using `newPlot` to `supplyAllDefaults`. However, it fails since the range doesn't seem to get set. Any idea how to solve this?
```javascript
    var supplyAllDefaults = require('../assets/supply_defaults');
    fit('should reverse yaxis if only images are present', function() {
        gd = {};
        gd.data = [{type: 'image', z: [[[255, 0, 0]]]}];
        gd.layout = {};
        supplyAllDefaults(gd);
        expect(gd._fullLayout.yaxis.range[0]).toBeGreaterThan(gd._fullLayout.yaxis.range[1]);
    });
```

Reply:
Never mind my last comment. I will check instead for `autorange === 'reversed'`. Commit underway

Reply:
Exactly, `supplyDefaults` won't give you the `range`, but assert `autorange` is 100% sufficient here.

Reply:
Done in 97a8d246f0025ae12204781d68582fc7235ff106